app/controllers/admin/vote_users_controller.rb

```diff
@@ -1,6 +1,5 @@
-class Admin::VoteUsersController < ApplicationController
-  skip_authorization
-  before_action :authorize
+class Admin::VoteUsersController < Admin::BaseController
+  authorize_resource class: false
```

Comment:
Yes, this is to separate a controller from a model, because this one would otherwise look for a `VoteUser`. With this solution you give permissions to `(:all, :vote_users)`.